src/features/surveys/components/surveyForm/SurveyTextBlock.tsx

```diff
@@ -0,0 +1,12 @@
+import { FC } from 'react';
+import { ZetkinSurveyTextElement } from 'utils/types/zetkin';
+
+export type SurveyTextBlockProps = {
+  element: ZetkinSurveyTextElement;
+};
+
+const SurveyTextBlock: FC<SurveyTextBlockProps> = ({ element }) => {
+  return <p>{element.text_block.content}</p>;
```

Comment:
For this we would normally use a `Typography` (which is just a themed `p` with some additional stuff).

Reply:
I might be wrong here, and the `variant="p"` is required, but I don't think so. I think it's the default value.

Reply:
Fixed in daaba32837bcb644a4a93be5f1b408a68781361e!